tests/e2e/e2e_setup_test.go

```diff
@@ -368,14 +412,20 @@ func (s *IntegrationTestSuite) configureChain(chainId string, validatorConfigs [
 	// If upgrade is skipped, we can use the chain initialization logic from
 	// current branch directly. As a result, there is no need to run this
 	// via Docker.
+	if !s.isFork {
+		s.forkHeight = 0
+	}
+
 	if s.skipUpgrade {
-		initializedChain, err := chain.Init(chainId, tmpDir, validatorConfigs, time.Duration(newChainConfig.votingPeriod))
+		initializedChain, err := chain.Init(chainId, tmpDir, validatorConfigs, time.Duration(newChainConfig.votingPeriod), s.forkHeight)
 		s.Require().NoError(err)
 		s.initializeChainConfig(&newChainConfig, initializedChain)
 		return
 	}
 
-	initResource, err := s.containerManager.RunChainInitResource(chainId, int(newChainConfig.votingPeriod), validatorConfigBytes, tmpDir)
+	s.forkHeight = s.forkHeight - forkHeightPreUpgradeOffset
+	fmt.Printf("FORK HEIGHT %v", s.forkHeight)
```

Comment:
let's use a proper log if this is useful or remove if not

Reply:
Oof I left this behind from debugging, thanks